api/src/main/java/org/geysermc/api/AuthType.java

```diff
@@ -0,0 +1,30 @@
+package org.geysermc.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum AuthType {
+    OFFLINE("offline"),
+    ONLINE("online"),
+    FLOODGATE("floodgate");
```

Comment:
Are these strings needed for the name? You can do `enumValue.name()`

Reply:
Not really, but I thought that it would be easier to use getName() instead of name().toLowerCase()